assets/src/components/Lightbox/Lightbox.js

```diff
@@ -28,13 +28,13 @@ export const Lightbox = ({ index, isOpen, items, onClose = () => {} }) => {
 
     const photoSwipe = new PhotoSwipe(photoSwipeElement, PhotoSwipeUI_Default, items, photoSwipeOptions);
 
-    photoSwipe.listen('gettingData', function (index, galleryItem) {
+    photoSwipe.listen('gettingData', (galleryItem) => {
```

Comment:
The Lightbox no longer works as expected if you remove the `index` param, even if it's unused... It's probably a function from the library